packages/main/test/pages/styles/TextArea.css

```diff
@@ -3,6 +3,10 @@ html,
     margin: 0;
 }
 
+span {
+    color: var(--sapTextColor);
```

Comment:
You can also consider using ui5-label, ui5-title instead if adding styles to h, span, then the theming will be out of the box